cloudscale/resource_cloudscale_network.go

```diff
@@ -17,6 +17,9 @@ func resourceCloudscaleNetwork() *schema.Resource {
 		Delete: resourceNetworkDelete,
 
 		Schema: getNetworkSchema(false),
+		Importer: &schema.ResourceImporter{
```

Comment:
This commit has a strange commit message. Maybe you want to reword it and force push 💥 ...